### PR TITLE
bump libocr ;rm P2P Networking V1 & libp2p

### PR DESCRIPTION
**Pull request by jmank88**
https://smartcontract-it.atlassian.net/browse/BCF-2708

Bump libocr, remove P2P Networking V1 and libp2p dependencies (~60!).

Requires:
- https://github.com/smartcontractkit/operator-ui/pull/61

### CODE REVIEW

File: core/services/ocrcommon/peer_wrapper.go [hunk 1]
```diff
@@ -139,70 +127,16 @@ func (p *SingletonPeerWrapper) peerConfig() (ocrnetworking.PeerConfig, error) {
...
-		PrivKey:         key.PrivKey,
-		Logger:          relaylogger.NewOCRWrapper(p.lggr, p.ocrCfg.TraceLogging(), func(string) {}),
-		// V1 config
-		V1ListenIP:                         config.V1().ListenIP(),
-		V1ListenPort:                       p2pPort,
-		V1AnnounceIP:                       v1AnnounceIP,
-		V1AnnouncePort:                     v1AnnouncePort,
-		V1Peerstore:                        peerStore,
-		V1DHTAnnouncementCounterUserPrefix: config.V1().DHTAnnouncementCounterUserPrefix(),
+		PrivKey: ed25519.PrivateKey(pk),
```
Comment (jmank88) [inline core/services/ocrcommon/peer_wrapper.go:138]: Is this right?

File: core/internal/testutils/configtest/v2/general_config.go [hunk 2]
```diff
@@ -34,6 +34,9 @@ func NewGeneralConfig(t testing.TB, overrideFn func(*chainlink.Config, *chainlin
 		},
 	}.New()
 	require.NoError(t, err)
+	if g.P2P().V1().Enabled() {
+		t.Skip("P2P.V1.Enabled") //TODO upgrade or remove these
```
Comment (jmank88) [inline core/internal/testutils/configtest/v2/general_config.go:38]: - [ ] upgrade or remove tests with `P2P.V1.Enabled`

File: core/internal/cltest/cltest.go [hunk 3]
```diff
@@ -324,6 +324,9 @@ const (
 func NewApplicationWithConfig(t testing.TB, cfg chainlink.GeneralConfig, flagsAndDeps ...interface{}) *TestApplication {
 	t.Helper()
 	testutils.SkipShortDB(t)
+	if cfg.P2P().V1().Enabled() {
+		t.Skip("P2P.V1.Enabled") //TODO upgrade or remove these
```
Comment (jmank88) [inline core/internal/cltest/cltest.go:328]: - [ ] upgrade or remove tests with `P2P.V1.Enabled`

File: integration-tests/types/config/node/core.go [hunk 4]
```diff
@@ -106,6 +106,7 @@ func WithOCR2() NodeConfigOpt {
 	}
 }
 
+// Deprecated
```
Comment (jmank88) [inline integration-tests/types/config/node/core.go:109]: Maybe nuke this?

File: core/config/toml/types.go [hunk 5]
```diff
@@ -868,19 +867,6 @@ type P2P struct {
 	V2 P2PV2 `toml:",omitempty"`
 }
 
-func (p *P2P) NetworkStack() ocrnetworking.NetworkingStack {
```
[the remark this answers is not in the document]
Comment (jmank88) [reply core/config/toml/types.go:897]: Well we were just inferring it before so there is no specific field to check. And we reject unrecognized config so we will already refuse to boot if they still have V1 configured.

**Review by samsondav — APPROVED** [review]

File: core/web/assets/index.html [hunk 6]
```diff
@@ -1 +1 @@
...
\ No newline at end of file
+<!doctype html><html><head><meta charset="UTF-8"/><meta name="viewport" content="width=device-width,initial-scale=1"/><title>Operator UI</title><link rel="shortcut icon" href="https://smartcontract.imgix.net/chainlink.ico?auto=compress%2Cformat"/><link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500" rel="stylesheet"/><link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet"/><link href="/favicon.ico" rel="shortcut icon" type="image/x-icon"/><title>Chainlink</title><script defer="defer" src="/assets/main.8f602c136d4004a835ee.js"></script></head><body><noscript>You need to enable JavaScript to run this app.</noscript><div id="root"></div></body></html>
```
[the remark this answers is not in the document]
Comment (jmank88) [reply core/web/assets/index.html:1]: There were problems with the fake assets that we used to embed (and then replace when actually building), so we decided to generate and commit the real ones instead. CI is already checking that generate is up to date, so piggy backing on that made it easy.

**Review by krehermann — APPROVED** [review]

**Pull request merged**